package.json

```diff
@@ -68,6 +68,7 @@
   "dependencies": {
     "babel": "^5.8.38",
     "babel-plugin-htmlbars-inline-precompile": "0.0.5",
+    "ember-cli-legacy-blueprints": "^0.1.0",
```

Comment:
typically we prefer this yes, as ember-cli being dependent on ^ may change over time. But this app appears to require it